js/views/settingsVw.js

```diff
@@ -758,6 +787,7 @@ module.exports = Backbone.View.extend({
     if(newAddress.name || newAddress.street || newAddress.city || newAddress.state || newAddress.postal_code) {
       if(!newAddress.name || !newAddress.street || !newAddress.city || !newAddress.state || !newAddress.postal_code){
         messageModal.show(window.polyglot.t('errorMessages.saveError'), window.polyglot.t('errorMessages.missingError'));
+        setTimeout(() => $(e.target).removeClass('loading'), 0);
```

Comment:
Since we're not in a deferred, just removing the loading class would be pointless since this handler is being executed before the body handler, so the body handler would put it right back. Thus, the setTimeout.

I'm very hesitant to stopPropogation, because there may legitimately be other pieces of code that rely on the event being propagated. Perhaps, there's a parent view that is targeting the event, etc...